doc/explanation/external-builder.md

```diff
@@ -1,5 +1,5 @@
 # External builder
 
-In order to pre-bootstrap LXD, Microk8s snaps to Juju, external VMs are spanwed and then run with
+In order to pre-bootstrap LXD, MicroK8s snaps to Juju; then, external VMs are spawned and ran with
```

Comment:
s/then, external VMs are spawned/external VMs are then spawned seems simpler grammar to me